### ISSUE TITLE
Folder drupal, README.md outdated and referred to archipelago-deployment NT live

### ISSUE THREAD

**Issue by giancarlobi** [esmero/archipelago-deployment-live#41]
@DiegoPino the README.md in folder drupal of this proyect (https://github.com/esmero/archipelago-deployment-live/blob/8fb1df98a2fe4cfbd11c2f4adb4c2e88f119fe03/drupal/README.md) is from and for archipelago-deployment (NOT -live) so all links here return 404. Probably to update or remove. Just a minor issue

**Comment by DiegoPino** [comment]
@giancarlobi true. We do not need that README there at all. The main repo readme does the work. I will correct (delete) ASAP. Thanks!
